Documentation/SwiftGenKit Contexts/MigrationGuide.md

```diff
@@ -1,9 +1,29 @@
-# Master
+# SwiftGen 6.0 Migration Guide
 
-*** Work In Progress — Will be consolidated once the release is ready ***
+## Changes for users
+
+### Strings
+
+The parser will no longer normalize string keys, which can lead to duplicate keys if your file contain similar keys but with different casing. For example, if your file contains:
+
+```
+"abcNews.something" = "foo";
+"ABCNews.somethingElse" = "bar";
+```
+
+SwiftGenKit will no longer consolidate these into one "abcNews" case. It is up to the user to fix this incorrect casing in their `strings` files.
+
+## Changes for template writers
+
+### XCAssets
+
+Groups now have an extra attribute `isNamespaced` that reflects the "provides namespace" setting in Xcode.
 
```

Comment:
Also from the change in StencilSwiftKit : we've migrated to "modes" (`:pretty` etc) instead of Bool param on `swiftIdentifier` and `snakeToCamelCase` (at least)

Reply:
(It's not really related to SwiftGenKit contexts, so not sure in which MG to put that, but since StencilSwiftKit doesn't have its own MG, and that's the place where we list things for template writers…)

Reply:
No we haven't. That issue is still open for discussion: https://github.com/SwiftGen/StencilSwiftKit/issues/100 (together with https://github.com/SwiftGen/StencilSwiftKit/issues/101).

Reply:
I was referring to https://github.com/SwiftGen/StencilSwiftKit/pull/61 which was introduced in StencilSwiftKit 2.1.0. if I'm not mistaken SwiftGen 5.x was originally shipped with StencilSwiftKit 2.0 so that change didn't make it to the 4.x -> 5.x MG, hence being interesting to report now?

Reply:
It was shipped in SwiftGen 5.1, quite a while ago. If you want, we can add it now to the 5.x migration guide, but I don't think we should add it to the 6.x MG. SwiftGen 5.1 is more than a year old now.

Reply:
We previously mentioned such changes in the general MG: https://github.com/SwiftGen/SwiftGen/blob/master/Documentation/MigrationGuide.md#some-swiftgen-specific-stencil-filters-evolved

Reply:
Right. Let's forget about it I guess then.